install/tests/Makefile

```diff
@@ -364,9 +380,12 @@ kots-install: version-flag = $(if $(version:-=),--app-version-label=$(version),)
 kots-install: preflight-flag = $(if $(preflights:true=),--skip-preflights,)
 kots-install: license-file = $(if $(license_community_$(channel)),$(license_community_$(channel)),"../licenses/$(channel).yaml")
 kots-install: install-kots-cli
-	kubectl kots remove ${app} -n gitpod --force --kubeconfig=${KUBECONFIG} || echo "No kots app existing, Installing"
-	$(MAKE) destroy-gitpod
-	export KUBECONFIG=${KUBECONFIG} && \
+	@echo "Cleaning up and re-installing Gitpod"
+	@kubectl kots remove ${app} -n gitpod --force --kubeconfig=${KUBECONFIG} || echo "No kots app existing, Installing"
+	@kubectl get secret -n gitpod https-certificates -o yaml > https-secret.yaml || echo "No existing certificate"
+	@$(MAKE) destroy-gitpod
+	@[[ -f https-secret.yaml ]] && kubectl apply -f https-secret.yaml || echo "No existing certificate"
```

Comment:
**issue(out-of-scope):** We're implementing optional behavior by doing `optional-command || echo "..."`; this works but when debugging failed CI jobs I've struggled to understand if these messages are indicating a problem. I would find it really helpful to amend these messages to indicate what's going on, if it's a problem, and if I need to do anything when encountering it.

Reply:
We can take this up in another improvement PR